src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Well, nothing ;) It's mostly a matter of personal style guide and the way you structure your files :)